Automattic/jetpack
# Staging Sites: Remove a filter that disables Jetpack staging mode for WPCOM staging sites

Fixes 7894-gh-Automattic/dotcom-forge.

## Proposed changes:

* Reverts changes introduced in https://github.com/Automattic/wpcomsh/pull/1314 (`wpcomsh`). The hook `jetpack_is_staging_site` is no longer invoked, as the only place where it was triggered is the deprecated function `is_staging_site`. The function was deprecated as part of https://github.com/Automattic/jetpack/pull/37023.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






> [!NOTE]
> The code is no longer used so there's no way to test it. Originally, we could test that `staging` badge was displayed. But following [this change](https://github.com/Automattic/jetpack/pull/37023/files#diff-028fe0b6a09c6ed2b0a61f21d8db4d21cf8ca35c8904f94f78503619455fe31fL1675-L1677), the badge is no longer rendered.
